modules/primer-avatars/package.json

```diff
@@ -17,15 +17,18 @@
     "url": "https://github.com/primer/primer-css/issues"
   },
   "scripts": {
+    "ava": "ava --verbose \"../../tests/modules/test-*.js\"",
     "build": "primer-module-build index.scss",
     "prepare": "npm run build",
     "lint": "stylelint **/*.scss -c .stylelintrc.json -s scss",
-    "test": "npm run build && npm run lint"
+    "test": "npm run build && npm run lint && npm run ava"
   },
   "dependencies": {
     "primer-support": "^4.1.0"
   },
   "devDependencies": {
+    "ava": "^0.21.0",
+    "glob": "^7.1.2",
```

Comment:
I found I needed these in the root package.json also should I remove it from here?